docs/source/protocol_v3.rst

```diff
@@ -419,9 +419,9 @@ time packets were previously specified for this session in the :ref:`session inf
       1 (int32) Number of time packets being sent
 
    Body:
-      <val> (float64) dt for the simulation
-      <val> (float64) Current time of the simulation
-      <val> (int64) Current integration step of the simulation
+      <val> (float64) Current integration time step (dt) for the simulation
+      <val> (float64) Current simulation time
+      <val> (int64) Current integration step counter of the simulation
```

Comment:
I understand the idea to make it clearer that this counts the steps but I think that "integration step" is the more common way to state this. Together with the int data type, it should be clear.
```suggestion
      <val> (int64) Current integration step of the simulation
```